test/monitoring/scrape_targets_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build aws packet
+// +build poste2e
+
+package monitoring
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"text/tabwriter"
+	"time"
+
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	retryInterval  = time.Second * 5
+	timeout        = time.Minute * 9
+	contextTimeout = 10
+)
+
+func testScrapeTargetRechability(t *testing.T, v1api v1.API) {
+	var w *tabwriter.Writer
+
+	if err := wait.PollImmediate(
+		retryInterval, timeout, getScrapeTargetRetryFunc(t, v1api, w),
+	); err != nil {
+		t.Errorf("%v", err)
+
+		// Finally print the table of all the targets that are down.
+		if err := w.Flush(); err != nil {
+			t.Errorf("error printing the unreachable targets: %v", err)
+		}
+	}
+}
+
+func getScrapeTargetRetryFunc(t *testing.T, v1api v1.API,
```

Comment:
Maybe we move `func() (done bool, err error)` to separate type, so we can have declaration in a single line? I don't like this formatting at all.

Reply:
done.